tests/js/spec/views/settings/organizationMembers/organizationMembersList.spec.jsx

```diff
@@ -268,13 +272,6 @@ describe('OrganizationMembers', function() {
   });
 
   it('can re-send invite to member', async function() {
-    const inviteMock = MockApiClient.addMockResponse({
```

Comment:
Would this work if we didn't mock `resendMemberInvite`?

Reply:
Fixed